### PR TITLE
Feature/profile cards

### PR DESCRIPTION
**Pull request by vasyldzyuba**

### CODE REVIEW

File: src/app/component/user/components/profile/profile-cards/profile-cards.component.html [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <div class="right-cards">
-  <div [ngStyle]="{'backgroundColor': activity.backgroundColor}"
-       class="card"
-       *ngFor="let activity of cardsData;">
-    <p class="cart-title">{{activity.title}}</p>
-    <p class="card-description">{{activity.description}}</p>
+  <div class="card"
+       *ngFor="let activity of facts;">
+    <p class="cart-title">Fact of the Day</p>
+    <p class="card-description">{{activity.content}}</p>
```
Comment (Gusvarov) [inline src/app/component/user/components/profile/profile-cards/profile-cards.component.html:5]: Please, add spaces between brackets

File: src/app/component/user/components/profile/profile-service/profile.service.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+})
+
+export class ProfileService {
+
+  constructor(private http: HttpClient) { }
+
+  private backEnd = environment.backendLink;
+
+  public getFactsOfTheDay(): Observable<CardModel> {
+    return this.http.get<CardModel>(`${this.backEnd}facts/dayFact/2?size=3`);
```
Comment (KimAT25) [inline src/app/component/user/components/profile/profile-service/profile.service.ts:18]: please not use magic numbers in the link

Comment (vasyldzyuba) [reply src/app/component/user/components/profile/profile-service/profile.service.ts]: Fixed

File: src/app/component/user/components/profile/profile-cards/profile-cards.component.html (hunk 1 above)
Comment (vasyldzyuba) [reply src/app/component/user/components/profile/profile-cards/profile-cards.component.html]: Fixed

**Pull request merged**